renderdoc/driver/gl/gl_resources.h

```diff
@@ -28,6 +28,8 @@
 #include "core/resource_manager.h"
 #include "driver/gl/gl_common.h"
 
+#define eGL_BGRA8_EXT		((GLenum)0x93A1)
```

Comment:
I've updated the headers - it seems like the GLES extension header wasn't committed before, so I've added it and added all the enums from there, including BGRA8_EXT.

You should be able to remove this now and then rebase your code onto master (it's best not to merge master since it will then create a merge commit).